collectors/channels_collector.go

```diff
@@ -416,8 +416,20 @@ func (c *ChannelsCollector) Collect(ch chan<- prometheus.Metric) {
 		anchorStateToString(lndclient.ForceCloseAnchorStateLost):      0,
 	}
 	for _, forceClose := range pendingChannelsResp.PendingForceClose {
+		var balance btcutil.Amount
+		switch forceClose.AnchorState {
+		case lndclient.ForceCloseAnchorStateRecovered:
+			balance = forceClose.RecoveredBalance
+
+		case lndclient.ForceCloseAnchorStateLimbo:
+			balance = forceClose.LimboBalance
+
+		default:
+			balance = forceClose.Capacity
```

Comment:
Isn't this quite wrong for non-chanchor channels? Shouldn't the default probably be `0` for non-anchor?

Reply:
Thanks for the thorough explanation on chat, and for the diff! Applied!